egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -90,23 +90,29 @@ def AddPermuteLayer(config_lines, name, input, column_map):
     return {'descriptor': '{0}_permute'.format(name),
             'dimension': input['dimension']}
 
-def AddAffineLayer(config_lines, name, input, output_dim, ng_affine_options = ""):
+def AddAffineLayer(config_lines, name, input, output_dim, ng_affine_options = "", max_change_per_component = 0.25):
```

Comment:
I notice that in different parts of the code, for different layer types, you have different values for max_change_per_component.  Can you give me some idea of how you tuned this?  Was this based just on WER, or did you also look at the diagnostics?